test/integration_test/storkctl_schedulepolicy_test.go

```diff
@@ -0,0 +1,196 @@
+//go:build integrationtest
+// +build integrationtest
+
+package integrationtest
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	storkv1 "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	"github.com/libopenstorage/stork/pkg/storkctl"
+	storkops "github.com/portworx/sched-ops/k8s/stork"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+	"io"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"os"
+	"testing"
+	"time"
+)
+
+func TestStorkCtlSchedulePolicy(t *testing.T) {
```

Comment:
Let's update the test IDs in test rail too with the results.
Most likely these tests must be already present, if not lets add it and put those IDs.

Reply:
I have added the test cases in test rail directory https://portworx.testrail.net/index.php?/suites/view/1&group_by=cases:section_id&group_order=asc&display_deleted_cases=0&group_id=9153
Have updated the code as well